src/overlays/actors/ovl_En_Time_Tag/z_en_time_tag.c

```diff
@@ -100,205 +113,230 @@ void func_80AC9FE4(EnTimeTag* this, PlayState* play) {
     if (ActorCutscene_GetCurrentIndex() != this->actor.cutscene) {
-        this->actionFunc = func_80ACA268;
+        this->actionFunc = EnTimeTag_SoaringEngraving_Wait;
         this->actor.textId = 0xC02;
         Item_Give(play, ITEM_SONG_SOARING);
     }
 }
 
-void func_80ACA184(EnTimeTag* this, PlayState* play) {
+void EnTimeTag_SoaringEngraving_StartCutscene(EnTimeTag* this, PlayState* play) {
     if (ActorCutscene_GetCurrentIndex() == 0x7C) {
         ActorCutscene_Stop(0x7C);
         ActorCutscene_SetIntentToPlay(this->actor.cutscene);
     } else if (ActorCutscene_GetCanPlayNext(this->actor.cutscene)) {
         ActorCutscene_Start(this->actor.cutscene, &this->actor);
-        this->actionFunc = func_80ACA12C;
+        this->actionFunc = EnTimeTag_SoaringEngraving_GetSong;
     } else {
         ActorCutscene_SetIntentToPlay(this->actor.cutscene);
     }
 }
 
-void func_80ACA208(EnTimeTag* this, PlayState* play) {
+void EnTimeTag_SoaringEngraving_SubsequentInteraction(EnTimeTag* this, PlayState* play) {
     if ((Message_GetState(&play->msgCtx) == TEXT_STATE_5) && Message_ShouldAdvance(play)) {
-        func_801477B4(play);
-        this->actionFunc = func_80ACA268;
+        Message_CloseTextbox(play);
+        this->actionFunc = EnTimeTag_SoaringEngraving_Wait;
     }
 }
 
-void func_80ACA268(EnTimeTag* this, PlayState* play) {
+void EnTimeTag_SoaringEngraving_Wait(EnTimeTag* this, PlayState* play) {
     if (Actor_ProcessTalkRequest(&this->actor, &play->state)) {
         if (this->actor.textId == 0) {
-            this->actionFunc = func_80ACA184;
+            this->actionFunc = EnTimeTag_SoaringEngraving_StartCutscene;
         } else {
-            this->actionFunc = func_80ACA208;
+            this->actionFunc = EnTimeTag_SoaringEngraving_SubsequentInteraction;
         }
     } else if ((this->actor.xzDistToPlayer < 100.0f) && Player_IsFacingActor(&this->actor, 0x3000, play) &&
-               (Flags_GetSwitch(play, ENTIMETAG_GET_SWITCHFLAG(&this->actor)) ||
+               (Flags_GetSwitch(play, TIMETAG_SOARING_GET_SWITCHFLAG(&this->actor)) ||
                 CHECK_QUEST_ITEM(QUEST_SONG_SOARING))) {
         this->actor.flags |= ACTOR_FLAG_1;
         func_800B8614(&this->actor, play, 110.0f);
     }
 }
 
-void func_80ACA348(EnTimeTag* this, PlayState* play) {
-    if (this->actor.home.rot.x > 0) {
-        this->actor.home.rot.x--;
+void EnTimeTag_Diary_AfterOcarina(EnTimeTag* this, PlayState* play) {
+    if (TIMETAG_DIARY_TIMER(&this->actor) > 0) {
+        TIMETAG_DIARY_TIMER(&this->actor)--;
         return;
     }
 
-    if (this->actor.home.rot.z != 0) {
-        func_80151938(play, 0x1230);
+    if (TIMETAG_DIARY_SONG(&this->actor) != TIMETAG_DIARY_SONG_EVAN_PART1) {
+        // TIMETAG_DIARY_SONG_EVAN_PART2
+        Message_ContinueTextbox(play, 0x1230);
     } else {
-        func_80151938(play, 0x122D);
+        // TIMETAG_DIARY_SONG_EVAN_PART1
+        Message_ContinueTextbox(play, 0x122D);
     }
```

Comment:
I find these comments rather awkward since it looks like they refer to the function call below rather than the whole block. Not sure what we can do about that, though, this is a common problem in the repo. Possibly just a blank line after?